dask/utils.py

```diff
@@ -441,7 +441,7 @@ def _skip_doctest(line):
     # NumPy docstring contains cursor and comment only example
     stripped = line.strip()
     if stripped == '>>>' or stripped.startswith('>>> #'):
-        return stripped
+        return line
```

Comment:
@mrocklin , any idea why this was so? The example in `np.tensordot` may have been the only one with `">>> #"` in the docstring...

Reply:
Goes back to https://github.com/dask/dask/pull/1669/files#diff-64b17ee9155433cad463c80a971a3223L547. I think if the CI is passing then we're OK with this change.